src/plat/pc99/linker.lds

```diff
@@ -27,10 +27,17 @@ nodeSkimScratchOffset = nodeSkimScratch - node_info;
 
 KERNEL_OFFSET = KLOAD_VADDR - KLOAD_PADDR;
 
+PHDRS {
+    headers PT_PHDR PHDRS ;
+    phys PT_LOAD FILEHDR PHDRS ;
+    boot PT_LOAD ;
+    virt PT_LOAD ;
+}
+
 SECTIONS
 {
     /* load kernel to 1M to avoid the famous IA-32 memory holes below */
-    . = KLOAD_PADDR;
+    . = KLOAD_PADDR + SIZEOF_HEADERS;
```

Comment:
This + SIZEOF_HEADERS allows the first section .phys to be placed directly after the ELF segment headers in the file.  Otherwise it would pad up to 4K alignment with zeroes.